website/docs/cdktf/release/upgrade-guide-v0-15.mdx

```diff
@@ -0,0 +1,104 @@
+---
+page_title: Upgrading to CDKTF Version 0.15
+description: >-
+  Minimum version of Terraform increased to 1.2, classes reanamed for C#, Java, and Go
+---
+
+# Upgrading to CDK for Terraform Version 0.15
+
+0.15 is primarily focused around CLI improvements targeted at a better Terraform Cloud integration.
+Because of this, there are only smaller breaking changes in this release. The biggest one is a refactoring of several Constructs in our core library.
+Some configuration classes were suffixed inconsistently with `Config`, `Props` or `Options` and were renamed to `Config` for consistency. These classes need to be used in C#, Java, and Go, therefore we have a breaking change in these languages.
+
+## Minimum version of Terraform increased to 1.2
+
+CDKTF is bumping the minimum supported version of Terraform from 1.0 to 1.2 starting from CDKTF version 0.15. This change is necessary as CDKTF 0.15 uses the Terraform CLI for planning and applying changes for Terraform Cloud and Terraform Enterprise instead of [the API](https://developer.hashicorp.com/terraform/cloud-docs/api-docs) used in version 0.14.
+
+[Terraform promises compatibility](https://developer.hashicorp.com/terraform/language/v1-compatibility-promises) between all v1.x releases, so there shouldn't be any issues with this change. The only changes between these versions are bug fixes, and new feature additions.
+
+Please open an issue if you encounter any issues or bugs with this change.
+
+## Renaming of `*Options` and `*Props` to `*Config`
+
+We had several exports where the configuration passed to constructs or functions was using a class with a different suffix then `Config`. If you are using a language like C#, Java, Python, or Go you might have needed to specify these classes in your CDKTF application. To simplify the usage we changed every suffix to be `Config`, so if you currently use one of the others please rename them to use the `Config` suffix. No options were changed in the process, only the names were aligned.
```

Comment:
this comment is naming Python (and we have no example) while all other mentions of languages above don't include Python – were there changes to Python? I suspect there were some, as you sometimes have to use some interface names for Python?

Reply:
Removing Python, I don't think one needs to specify them in python